### PR TITLE
Add a snapshot test for `Path` SVG layout

### PR DESCRIPTION
**Pull request by MaxDesiatov**
Depends on #253.

This adds a dependency on the [SnapshotTesting](https://github.com/pointfreeco/swift-snapshot-testing) library, which allows testing our SVG layout algorithm end-to-end. We use the `--screenshot` flag of Chromium-based browsers (MS Edge in this case) to produce a PNG snapshot of a view rendered with `StaticHTMLRenderer`.

This test works only on macOS for now due to its dependency on `NSImage`, but that should be fine as we'd expect the same SVG output to be rendered in the same way on all platforms.

### CODE REVIEW

File: Sources/TokamakCore/Shapes/Path/PathLayout.swift [hunk 1]
```diff
@@ -12,185 +12,11 @@
...
 // limitations under the License.
 //
-//  Created by Carson Katri on 06/28/2020.
+//  Created by Max Desiatov on 20/06/2021.
 //
 
 import Foundation
 
-/// The outline of a 2D shape.
-public struct Path: Equatable, LosslessStringConvertible {
```
Comment (MaxDesiatov) [inline Sources/TokamakCore/Shapes/Path/PathLayout.swift:21]: Moved to a separate `Path.swift` file to fix linter warnings.

File: Tests/TokamakTests/ReconcilerStressTests.swift [hunk 2]
```diff
@@ -40,20 +40,9 @@ final class ReconcilerStressTests: XCTestCase {
     let renderer = TestRenderer(SpookyHanger())
     let root = renderer.rootTarget
 
-    return
```
Comment (MaxDesiatov) [inline Tests/TokamakTests/ReconcilerStressTests.swift:43]: This test was invalid, but the `return` statement made this undiscoverable. Cleaning that up and simplifying the test to actually pass.

File: Sources/TokamakCore/Shapes/Path/Path.swift [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Carson Katri on 06/28/2020.
+//
+
+import Foundation
+
+/// The outline of a 2D shape.
+public struct Path: Equatable, LosslessStringConvertible {
```
Comment (MaxDesiatov) [inline Sources/TokamakCore/Shapes/Path/Path.swift:21]: This is a smaller version of the original `Path.swift` that satisfies linter's file length constraints.

**Review by foscomputerservices — APPROVED** [review]
Nice!

**Pull request merged**